compiler/cpp/src/thrift/parse/t_field.h

```diff
@@ -68,7 +68,7 @@ class t_field : public t_doc {
 
   int32_t get_key() const { return key_; }
 
-  enum e_req { T_REQUIRED, T_OPTIONAL, T_OPT_IN_REQ_OUT };
+  enum e_req : unsigned { T_REQUIRED, T_OPTIONAL, T_OPT_IN_REQ_OUT };
```

Comment:
Got it. Should I add a new patch to this pull request or squash them into one patch on this pull request?